src/router.js

```diff
@@ -13,12 +19,22 @@ const routes =[
         name: 'contractor-home',
         component: ContractorHome,
     } ,
+    {
+        path: '/bidderHome',
```

Comment:
path should be kebab-case _bidderHome_ --> _bidder-home_ or simplified to just _bidder_